bigframes/core/groupby/__init__.py

```diff
@@ -91,8 +92,31 @@ def __getitem__(
 
         bad_keys = [key for key in keys if key not in self._block.column_labels]
 
-        if len(bad_keys) > 0:
-            raise KeyError(f"Columns not found: {str(bad_keys)[1:-1]}")
+        # Raise a KeyError message with the possible correct key(s)
+        if len(bad_keys) == 1:
```

Comment:
I think you might be able to combine these two cases and avoid the if statement.